golang/Go/parser/go_parser_base.go

```diff
@@ -2,7 +2,6 @@ package parser
 
 import (
 	"strings"
-
```

Comment:
I never understood why people have this blank line between "system" packages, and "local". Is it a "Go" requirement??? For God's sake, a package is a package! Stop this needless blank line crap. There are *so* many things wrong with Go in forcing formatting (requiring "{" to be in hanging style), we need to stop adding syntax that adds to the confusion.